CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+
+## Contributing
```

Comment:
A lot of content here only applies to the Python package. Could copy this [one](https://github.com/jackyho112/DSCI_522_Group405/blob/master/CONTRIBUTING.md) instead?

Reply:
Done!